conf/evolutions/default/59.sql

```diff
@@ -7,6 +7,43 @@ CREATE TABLE `system_settings` (
   PRIMARY KEY (`id`),
   UNIQUE INDEX `id_UNIQUE` (`id` ASC));
 
+
+ALTER TABLE system_settings
+ADD COLUMN description VARCHAR (300);
+
+UPDATE system_settings SET description =
+'This setting would allow user to add multiple chief complaints instead of just one'
+WHERE id = 1;
+
+UPDATE system_settings SET description =
+'This is a compulsory setting and cannot be unselected. '
+WHERE id = 2;
+
+UPDATE system_settings SET description =
+'Medical photo tab is a compulsory setting and cannot be unselected. '
+WHERE id = 3;
+
+UPDATE system_settings SET description =
+'This setting must be selected if medical HPI consolidation is to be permitted. '
+WHERE id = 4;
+
+UPDATE system_settings SET description =
+'This is a compulsory setting and cannot be unselected. It makes sure that the metric system is being used '
+WHERE id = 5;
+
+UPDATE system_settings SET description =
+'This setting must be selected, if you want to apply filter on mission countries. '
+WHERE id = 6;
+
+UPDATE system_settings SET description =
+'This setting must be selected if you want to allow the research only option. '
+WHERE id = 7;
+
+UPDATE system_settings SET description =
+'This setting must be selected to prompt for diabetes. '
+WHERE id = 8;
+
+
 # --- !Downs
```

Comment:
Hey @kratikaswami. Everything looks pretty good here, just some small things to tweak up.

The above edit to Evolution 59 looks pretty good, but there are a few things to take care of:

1. These changes should be in their own evolution file. 

The idea is that any existing femr install can have its code base updated, and it will automatically run the evolutions to change the database to how the code will expect it to be. In this case, there are femr setups that are already past evolution 59. In order to get these new changes, those systems will have to run the downs on every evolution they've already run (we are at 86) down to evolution 59. When this process is done, evolution 59 through 86 will have to run their ups again. This will cause some data loss as existing tables and columns will have to be removed during the downs and re-created with no data in the ups back to 86.

All you need to do is create a new evolution and copy/past the new sql to the new evolution (87.sql).

2. Some of the descriptions in this file can also be tweaked. See the Jira issue for this task (https://teamfemr.atlassian.net/browse/FEMR-126) for brief descriptions of what the settings do.

Also, none of the settings are compulsory or required to be checked. Feel free to tweak the descriptions on the Jira task as you see fit and let us know if you have any questions regarding this.